record-and-playback/core/lib/recordandplayback/generators/audio_processor.rb

```
@@ -53,37 +63,11 @@ def self.process(archive_dir, file_basename)
-        audio_inputs = []	
-        audio_inputs << @audio_file	
-        audio_inputs << BigBlueButton::EDL::Audio.render(deskshare_audio_edl, deskshare_dir)	
-
-        @audio_file = BigBlueButton::EDL::Audio.mixer(audio_inputs, mixed_dir)	
-      else
-        BigBlueButton.logger.info("AudioProcessor.process: no Deskshare audio to process.")	
-      end
+      @audio_file = BigBlueButton::EDL::Audio.render(audio_edl, File.join(target_dir, 'recording'))
 
       ogg_format = {
         :extension => 'ogg',
```

RuboCop (1.85.1)

[convention] Use the new Ruby 1.9 hash syntax.

(Style/HashSyntax)